TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/integ_test/integ_test/test_cases/ma_argo_test_base.py

```diff
@@ -104,7 +105,8 @@ def prepare_workflow_snapshot_and_migration_config(self):
         snapshot_and_migration_configs = [{
             "migrations": [{
                 "metadata": {
-                    "from_snapshot": None
+                    "from_snapshot": None,
+                    "otel_endpoint": OTEL_COLLECTOR_ENDPOINT
```

Comment:
Sorry if  this contradicts anything that I've said before - but shouldn't there be just one collector endpoint for the whole system (workflow)?  If that's the case, we'll just push it down through the workflow with each process.
I don't think that there's anything necessarily to change right now for this PR since https://github.com/opensearch-project/opensearch-migrations/pull/1633 has work to send configurations through different layers much more elegantly.  We'd just need to update where we set the value in the config...

But should we even require setting the value within the config or should it come from a configmap that was set at helm install-time (and available to this test).